### ISSUE TITLE
Next 13.4 (App Router)

### ISSUE THREAD

**Issue by berkaycirak** [glitch-txs/multi-chain-wallets-modal#1]
It looks pretty well, will you also add app router for that project @glitch-txs ?

**Comment by glitch-txs** [comment]
Thanks a lot 🙏 ideally I'd like to make it framework agnostic, so it should support app dir, it still needs a lot of work though, I'm kind of doing while learning 😅

**Comment by berkaycirak** [comment]
By the way I am trying to do a project where there is a wallet connect popup also. There will be three buttons which are Metamask Connect, Coinbase Connect and Walletconnect Connect. I have seen your project like that but I am not sure how to do that. I will use web3modal library for walletconnect option but for others what should I use? (I know web3modal provides Metamask connection but I want separately a metamask button instead of walletconnect modal)

**Comment by glitch-txs** [comment]
yes, web3modal uses wagmi as well, you can also use wagmi alone to get the providers (they call them connectors)

**Comment by berkaycirak** [comment]
Thanks a lot
